addons/web/static/src/core/popover/popover_controller.js

```diff
@@ -55,8 +56,8 @@ export class PopoverController extends Component {
     onClickAway(ev) {
         const target = ev.composedPath()[0];
         if (this.props.closeOnClickAway(target) && !this.isInside(target)) {
+            this.props.onClickAway?.(ev);
```

Comment:
maybe rename into `onWillCloseOnClickAway`? It's a long name, but my problem with `onClickAway` is that isn't called all the time "on click away"